libraries/rush-lib/src/utilities/Npm.ts

```diff
@@ -50,8 +50,9 @@ export class Npm {
           console.log(`No version is found for ${packageName}`);
         }
       }
-    } catch (error) {
-      if ((error as Error).message.indexOf('npm ERR! 404') >= 0) {
+    } catch (e) {
+      const error: Error = e;
```

Comment:
Does this error have a `code` property that we can check instead?

Reply:
Sadly no, it didn't provides us any useful information